### PR TITLE
ipn/ipnlocal: rename AuthURL fields for clarity

### PR DESCRIPTION
**Pull request by andrew-d**
Updates #cleanup

Change-Id: Id5d9d6c6a4e1ccf0525881b939198be493b4ff2d

### CODE REVIEW

File: ipn/ipnlocal/local.go [hunk 1]
```diff
@@ -2261,7 +2261,7 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 			ini.SessionID = sessionID
 			ini.State = ptr.To(b.state)
 			if b.state == ipn.NeedsLogin {
-				ini.BrowseToURL = ptr.To(b.authURLSticky)
+				ini.BrowseToURL = ptr.To(b.authURL)
```
Comment (maisem) [inline ipn/ipnlocal/local.go:2264]: should this be `authURLToPop`? or do you want to keep existing behavior in this PR and then fix in a follow up?

Comment (maisem) [reply ipn/ipnlocal/local.go:2264]: can also do 2 commits

File: ipn/ipnlocal/local.go [hunk 2]
```diff
@@ -4204,7 +4204,7 @@ func (b *LocalBackend) enterStateLockedOnEntry(newState ipn.State) {
 			b.logf("Reconfig(down): %v", err)
 		}
 
-		if authURL == "" {
+		if authURLToPop == "" {
```
Comment (maisem) [inline ipn/ipnlocal/local.go:4207]: i think this should also be authURL

**Comment by andrew-d** [comment]
Done elsewhere

**Pull request closed without merging**